### DISCUSSION TITLE
Search Results & Filters

### DISCUSSION THREAD

**Discussion by ellanrk** [microsoft-search/pnp-modern-search#3536] [Q&A] [answered]
My search results displaying tons of content from sites. But when I configure search filters, it's not displaying full list of sites (combo box) from which users can select the specific site to trim the results.

**Comment by kasperbolarsen** [comment, marked as answer]
That is a known limitation of the API, neither the total count nor all the refiners are returned when the number of refiners is high.  You will be able to find multiple reports/issues in this repo about that issue.
